includes/class-sensei.php

```diff
@@ -397,6 +397,9 @@ public function initialize_global_objects() {
 		$this->enrolment_scheduler = Sensei_Enrolment_Job_Scheduler::instance();
 		$this->enrolment_scheduler->init();
 
+		// Onboarding Wizard.
+		$this->onboarding = new Sensei_Onboarding();
```

Comment:
Oh, now that I saw that you put the onboarding out of the `if`. Is it only for tests purpose? If yes, maybe should we do it differently? Else, I'll refactor the test in the other PR to use it.

Reply:
This is needed for the REST API, because those requests are not invoking the WP-Admin screen.

Reply:
Oh, perfect! Tomorrow I'll refactor the tests in the other PR to use like this =)